### ISSUE TITLE
RfC: Change QuantityType serialisation for degree celsius

### ISSUE THREAD

**Issue by kaikreuzer** [eclipse/smarthome#5669]
I would like to suggest changing the string serialisation for degree celsius from the special character `℃` to `°C` (two separate characters).

When working with such units, I noticed that:
- many fonts either do not support `℃` at all or render it in some weird fashion
- Many Text-to-speech engines cannot cope with `℃`, while they correctly process `°C`.
- as `℃` is not on the keyboard, people won't use it themselves (in rules, sitemaps, etc.) and thus we will always have a mix which might result in issues (e.g. when trying to do string comparison).

TL;DR: I think our life would be easier if we avoid using the very uncommon char `℃`.

**Comment by J-N-K** [comment]
Full Ack.

**Comment by kaikreuzer** [comment]
@htreu Could you please comment whether this is an easy change?

**Comment by maggu2810** [comment]
I have not a strong opinion here? I like the usage of the respective unicode character the same as I prefer `²` and `³` instead of `^2` and `^3`.
If a font does not support that character I would change the font...
TTS, okay, this is a good point.
Missing key on the keyboard, another point why the normal user will not use it in its rule.

So, okay for me to change WRT to the last points, but if we keep it as it is its also okay for me. Perhaps there is a way we could map it internally so both are handled correctly.

**Comment by kaikreuzer** [comment]
Yes, I agree that both should continue to work - my main wish is to change the serialisation to `°C` as this stringified version is what ends up in UIs, TTS and in any other place.
